diff -aurN ardour-0.9beta5.orig/libs/ardour/configure.ac ardour-0.9beta5/libs/ardour/configure.ac
--- ardour-0.9beta5.orig/libs/ardour/configure.ac	2003-10-10 22:33:23.000000000 +0200
+++ ardour-0.9beta5/libs/ardour/configure.ac	2003-10-11 13:05:34.769207648 +0200
@@ -120,11 +120,6 @@
 CFLAGS="$CFLAGS $GLIB_CFLAGS $JACK_CFLAGS $LRDF_CFLAGS $XML_CFLAGS $CONFIGDIR $DATADIR"
 CXXFLAGS="$CXXFLAGS $GLIB_CFLAGS $JACK_CFLAGS $LRDF_CFLAGS $XML_CFLAGS $CONFIGDIR $DATADIR"
 
-dnl clean up those argument lists a bit ...
-AC_UNIQUIFY_FIRST([$CXXFLAGS],[CXXFLAGS])
-AC_UNIQUIFY_FIRST([$CFLAGS],[CFLAGS])
-AC_UNIQUIFY_LAST([$LIBS],[LIBS])
-
 AC_OUTPUT(Makefile 
           intl/Makefile 
           po/Makefile.in 
