--- ardour-0.9beta18/libs/pbd/aclocal.m4.orig	2004-07-06 00:29:18.000000000 +0200
+++ ardour-0.9beta18/libs/pbd/aclocal.m4	2004-07-09 00:08:45.500025552 +0200
@@ -866,49 +866,16 @@
 
 if test "$target_cpu" = "powerpc"; then
   AC_DEFINE(POWERPC, 1, "Are we running a ppc CPU?")
-  altivecLinux=
-  if test "$?" = "0"; then
-    AC_DEFINE(HAVE_ALTIVEC_LINUX, 1, "Is there Altivec Support ?")
-    if test "$gcc_major_version" = "3"; then
-dnl -mcpu=7450 does not reliably work with gcc 3.*
-      OPT_FLAGS="-D_REENTRANT -O2 -mcpu=7400 -maltivec -mabi=altivec"
-    else
-      OPT_FLAGS="-D_REENTRANT -O3 -mcpu=7400"
-    fi
-  else
-    OPT_FLAGS="-D_REENTRANT -O3 -mcpu=750 -mmultiple"
-  fi
-  OPT_FLAGS="$OPTIM_FLAGS -mhard-float -mpowerpc-gfxopt"
+  OPT_FLAGS="$CXXFLAGS -D_REENTRANT"
 elif echo $target_cpu | grep "i*86" >/dev/null; then
-  cat /proc/cpuinfo | grep mmx >/dev/null
-  if test $? = 0; then
-    mmx="-mmmx"
-  fi
-  cat /proc/cpuinfo | grep sse >/dev/null
-  if test $? = 0; then
-    sse="-msse -mfpmath=sse"
-  fi
-  cat /proc/cpuinfo | grep 3dnow >/dev/null
-  if test $? = 0; then
-    dreidnow="-m3dnow"
-  fi
   AC_DEFINE(x86, 1, "Nope its intel")
-  if test "$target_cpu" = "i586"; then
-    OPT_FLAGS="-DREENTRANT -O3 -march=i586 -fomit-frame-pointer -ffast-math -fstrength-reduce -fmove-all-movables"
-  elif test "$target_cpu" = "i686"; then
-    OPT_FLAGS="-D_REENTRANT -O3 -march=i686 -fomit-frame-pointer -ffast-math -fstrength-reduce -fmove-all-movables"
-    if test "$gcc_major_version" = "3"; then
-      OPT_FLAGS="$OPTIM_FLAGS $mmx $sse $dreidnow"
-    fi
-  else
-    OPT_FLAGS="-D_REENTRANT -O3 -fomit-frame-pointer -ffast-math -fstrength-reduce -fmove-all-movables"
-  fi
+  OPT_FLAGS="$CXXFLAGS -D_REENTRANT -ffast-math -fstrength-reduce -fmove-all-movables"
 fi
 
 # LARGEFILE_FLAGS="`getconf LFS_CFLAGS`"
 LARGEFILE_FLAGS="-D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE"
 
-CXXFLAGS="-g -D_REENTRANT $LARGEFILE_FLAGS"
+CXXFLAGS="$CXXFLAGS -D_REENTRANT $LARGEFILE_FLAGS"
 if test x"$GXX" = xyes ; then
    CXXFLAGS="$CXXFLAGS -Wall"
 fi
