--- ardour-0.9beta2/aclocal.m4.orig	2003-06-29 02:45:29.000000000 +0200
+++ ardour-0.9beta2/aclocal.m4	2003-07-15 00:02:13.000000000 +0200
@@ -806,19 +806,19 @@
 
 ARDOUR_TOP=`cd $ARDOUR_TOP; pwd`
 
-export LIBDIRS="sigc++ pbd soundtouch midi++ ardour gtkmm gtkmmext gtk-canvas"
+export LIBDIRS="pbd soundtouch midi++ ardour gtkmmext"
 
 ACLOCAL_FLAGS="-I $ARDOUR_TOP/aclocal"
 ACLOCAL_AMFLAGS="-I $ARDOUR_TOP/aclocal `if test -d m4 ; then echo -I m4; fi`"
 AC_SUBST(ACLOCAL_FLAGS)
 AC_SUBST(ACLOCAL_AMFLAGS)
 
-BASE_LIBS="$ARDOUR_TOP/libs/midi++/libmidipp.a $ARDOUR_TOP/libs/pbd/libpbd.a $ARDOUR_TOP/libs/sigc++/sigc++/libsigc.a"
-BASE_INCLUDES="-I$ARDOUR_TOP/libs/sigc++ -I$ARDOUR_TOP/libs/sigc++/sigc++/config -I$ARDOUR_TOP/libs/pbd -I$ARDOUR_TOP/libs/midi++ -I$ARDOUR_TOP/libs/ardour -I$ARDOUR_TIP/libs/soundtouch"
-GUI_LIBS="$ARDOUR_TOP/libs/gtkmmext/libgtkmmext.a $ARDOUR_TOP/libs/midi++/libmidipp.a $ARDOUR_TOP/libs/gtk-canvas/libgtk-canvas.a $ARDOUR_TOP/libs/gtkmm/src/gtk--/libgtkmm.a $ARDOUR_TOP/libs/gtkmm/gdk--/gdk--/libgdkmm.a"
-GUI_INCLUDES="-I$ARDOUR_TOP/libs/ -I$ARDOUR_TOP/libs/gtk-canvas -I$ARDOUR_TOP/libs/gtk-canvas/gtk-canvas -I$ARDOUR_TOP/libs/gtkmmext -I$ARDOUR_TOP/libs/gtkmm/gdk-- -I$ARDOUR_TOP/libs/gtkmm/src -I$ARDOUR_TOP/libs/gtkmm"
-CFLAGS="$CFLAGS -I$ARDOUR_TOP/libs/sigc++ -I$ARDOUR_TOP/libs/sigc++/sigc++/config -I$ARDOUR_TOP/libs/pbd -I$ARDOUR_TOP/libs/midi++ -I$ARDOUR_TOP/libs/ardour -I$ARDOUR_TOP/libs/soundtouch"
-CXXFLAGS="$CXXFLAGS -I$ARDOUR_TOP/libs/sigc++ -I$ARDOUR_TOP/libs/sigc++/sigc++/config -I$ARDOUR_TOP/libs/pbd -I$ARDOUR_TOP/libs/midi++ -I$ARDOUR_TOP/libs/ardour -I$ARDOUR_TOP/libs/soundtouch"
-LIBS="$LIBS $ARDOUR_TOP/libs/soundtouch/libsoundtouch.a $ARDOUR_TOP/libs/midi++/libmidipp.a $ARDOUR_TOP/libs/pbd/libpbd.a $ARDOUR_TOP/libs/sigc++/sigc++/libsigc.a"
+BASE_LIBS="$ARDOUR_TOP/libs/midi++/libmidipp.a $ARDOUR_TOP/libs/pbd/libpbd.a `sigc-config --libs`"
+BASE_INCLUDES="`sigc-config --cflags` -I$ARDOUR_TOP/libs/pbd -I$ARDOUR_TOP/libs/midi++ -I$ARDOUR_TOP/libs/ardour -I$ARDOUR_TIP/libs/soundtouch"
+GUI_LIBS="$ARDOUR_TOP/libs/gtkmmext/libgtkmmext.a $ARDOUR_TOP/libs/midi++/libmidipp.a -lgtk-canvas `gtkmm-config --libs`"
+GUI_INCLUDES="-I$ARDOUR_TOP/libs/ -I$ARDOUR_TOP/libs/gtkmmext `gtkmm-config --cflags`"
+CFLAGS="$CFLAGS `sigc-config --cflags` -I$ARDOUR_TOP/libs/pbd -I$ARDOUR_TOP/libs/midi++ -I$ARDOUR_TOP/libs/ardour -I$ARDOUR_TOP/libs/soundtouch"
+CXXFLAGS="$CXXFLAGS `sigc-config --cflags` -I$ARDOUR_TOP/libs/pbd -I$ARDOUR_TOP/libs/midi++ -I$ARDOUR_TOP/libs/ardour -I$ARDOUR_TOP/libs/soundtouch"
+LIBS="$LIBS $ARDOUR_TOP/libs/soundtouch/libsoundtouch.a $ARDOUR_TOP/libs/midi++/libmidipp.a $ARDOUR_TOP/libs/pbd/libpbd.a `sigc-config --libs`"
 ])
 
--- ardour-0.9beta2/libs/configure.ac.orig	2003-06-24 05:11:36.000000000 +0200
+++ ardour-0.9beta2/libs/configure.ac	2003-07-15 00:03:21.000000000 +0200
@@ -5,10 +5,10 @@
 
 AM_BUILD_ENVIRONMENT
 
-AC_CONFIG_SUBDIRS(sigc++ pbd soundtouch midi++ ardour)
+AC_CONFIG_SUBDIRS(pbd soundtouch midi++ ardour)
 
 if test x$WITH_GTK_GUI != x ; then
-   AC_CONFIG_SUBDIRS(gtkmm gtkmmext gtk-canvas)
+   AC_CONFIG_SUBDIRS(gtkmmext)
 fi
 
 if test x$WITH_VST_WIN != x ; then
--- ardour-0.9beta2/gtk_ardour/editor_audiotrack.cc.orig	2003-06-24 05:11:35.000000000 +0200
+++ ardour-0.9beta2/gtk_ardour/editor_audiotrack.cc	2003-07-15 00:57:21.000000000 +0200
@@ -1,11 +1,12 @@
-#include <ardour/location.h>
-#include <ardour/diskstream.h>
-
 #include "editor.h"
 #include "audio_time_axis.h"
 #include "regionview.h"
 #include "selection.h"
 
+/* must be after Gtk-- includes - LOCK define conflict */
+#include <ardour/location.h>
+#include <ardour/diskstream.h>
+
 using namespace ARDOUR;
 
 void
--- ardour-0.9beta2/gtk_ardour/selection.cc.orig	2003-06-24 05:11:35.000000000 +0200
+++ ardour-0.9beta2/gtk_ardour/selection.cc	2003-07-15 01:12:34.000000000 +0200
@@ -22,8 +22,6 @@
 #include <sigc++/bind.h>
 #include <pbd/error.h>
 
-#include <ardour/route.h>
-
 #include "regionview.h"
 #include "selection.h"
 #include "selection_templates.h"
@@ -31,6 +29,9 @@
 
 #include "i18n.h"
 
+/* must be after Gtk-- includes - LOCK define conflict */
+#include <ardour/route.h>
+
 using namespace ARDOUR;
 using namespace SigC;
 
