diff -dur -x '*~' Ardour-5.12.0.orig/gtk2_ardour/transcode_ffmpeg.cc Ardour-5.12.0/gtk2_ardour/transcode_ffmpeg.cc
--- Ardour-5.12.0.orig/gtk2_ardour/transcode_ffmpeg.cc	2017-09-15 22:56:22.000000000 +0200
+++ Ardour-5.12.0/gtk2_ardour/transcode_ffmpeg.cc	2018-10-23 19:27:09.621396487 +0200
@@ -60,10 +60,6 @@
 		                             "The tools are included with the %1 releases from ardour.org "
 		                             "and also available with the video-server at http://x42.github.com/harvid/\n"
 		                             "\n"
-		                             "Important: the files need to be installed in $PATH and named ffmpeg_harvid and ffprobe_harvid.\n"
-		                             "If you already have a suitable ffmpeg installation on your system, we recommend creating "
-		                             "symbolic links from ffmpeg to ffmpeg_harvid and from ffprobe to ffprobe_harvid.\n"
-		                             "\n"
 		                             "see also http://manual.ardour.org/video-timeline/setup/"),
 		                           PROGRAM_NAME)
 		        << endmsg;
diff -dur -x '*~' Ardour-5.12.0.orig/libs/ardour/video_tools_paths.cc Ardour-5.12.0/libs/ardour/video_tools_paths.cc
--- Ardour-5.12.0.orig/libs/ardour/video_tools_paths.cc	2017-09-15 22:56:22.000000000 +0200
+++ Ardour-5.12.0/libs/ardour/video_tools_paths.cc	2018-10-23 19:26:43.792949900 +0200
@@ -166,7 +166,7 @@
 	_ffprobe_exe = X_("");
 
 	std::string ff_file_path;
-	if (find_file (Searchpath(Glib::getenv("PATH")), X_("ffmpeg_harvid"), ff_file_path)) {
+	if (find_file (Searchpath(Glib::getenv("PATH")), X_("ffmpeg"), ff_file_path)) {
 		_ffmpeg_exe = ff_file_path;
 	}
 #ifdef PLATFORM_WINDOWS
